### ISSUE TITLE
Step 7 - no out dir

### ISSUE THREAD

**Issue by wikidude** [s9roll7/ebsynth_utility#20]
Hello,

I have been able to execute steps 1-5 on Linux. I deferred the full project folder to a windows machine and executed the indications from step 6 (running the .ebs file). The result was multiple out-00xxx folders, one for each keyframe. I then proceeded to move the project folder back to linux to run step 7 in stable diffusion. The folder path is the same as the original folder, yet step 7 results in 'no out dir' which makes me think the recursive directory search does not work on a linux filesystem or something.

Error:
blend_rate: 1
export_type: mp4
fps: 30.0
-1
no out dir

Would be great if the concatenation worked under Linux, even if the EbSynth process has to be deferred to a windows machine which returns the project folder.

Project folder structure after step 6:

![cz437](https://user-images.githubusercontent.com/28767197/224728889-68e9a92e-bc12-4954-8305-547cacc5fe4e.png)

Of course, there is always the option I did something wrong, in which case, I'm just dumb and you can close the issue. :)
Great work, btw! The stability is pretty good. The only thing missing is being able to use the `img2img alternative test` script to feed the keyframe image as input noise. That would help a lot in stability and consistency of keyframes output from img2img.

[Edit] Installing SD on the Windows machine and your extension, then just running step 7 on the same folder worked perfectly

**Comment by s9roll7** [comment]
Thanks for the report.
I just noticed that issue a few minutes ago and fixed it.
Give it a try.

**Comment by wikidude** [comment]
If that pattern changed in https://github.com/s9roll7/ebsynth_utility/commit/fae34f7ec6827af989079a8e0748eee9d5695633 is supposed to be regex, I think the correct format would be:
`.*[\\\/]out\-([0-9]+)[\\\/]`
![image](https://user-images.githubusercontent.com/28767197/224770959-ac1f7ab3-f8d9-4faf-b3e0-1954507de64e.png)
as the current `.*[\\/]out\-([0-9]+)[\\/]` would fail due to an unescaped character `/`
![image](https://user-images.githubusercontent.com/28767197/224771070-639c959d-9a22-4d1b-81b2-f8d1ad182b58.png)

Tested on linux with `.*[\\\/]out\-([0-9]+)[\\\/]` in line 41 of stage7.py. Restarted SD and it works correctly now (please make sure it would still work on Windows with this change)
![image](https://user-images.githubusercontent.com/28767197/224770604-14ad9a98-f9fc-493f-b129-1259c7f88986.png)

**Comment by s9roll7** [comment]
Thank you.I've fixed it.
It worked fine in windows environment.

**Comment by zaapas** [comment]
same error on windows today

**Comment by JumboSimp** [comment]
Also just got this error; was able to export fine when I first used the software, about a month ago to this date.

**Comment by chlowden** [comment]
I too have the same error with Automatic 1111 in linux.

**Comment by Vansaska** [comment]
I am getting this error on Windows

**Comment by chlowden** [comment]
It would be nice for it to work. Instead I import the image sequence into Davinci resolve and do a timewarp.
